crux-mir/report-coverage/src/main.rs

```diff
@@ -612,6 +662,25 @@ impl Reporter {
         }
     }
 
+    fn coverage_stats(&self, fun: &str, called: bool, branch_seen: usize, branch_tot: usize) {
+        use std::io::Write;
+        use termcolor::{Color, ColorSpec, StandardStream, WriteColor};
+
+
+        let call_stat = if called { "✅" } else { "❌" }; 
+        let mut stdout = StandardStream::stdout(self.color_choice);
+        stdout.set_color(ColorSpec::new().set_fg(None)).unwrap();
+        let perc = if branch_tot == 0 { if called { 100 } else { 0 } } else { 100 * branch_seen / branch_tot }; 
```

Comment:
I find this line of code somewhat hard to read since it packs multiple `if` expressions on the same line. How about splitting this up across multiple lines, e.g.,

```suggestion
        let perc =
            if branch_tot == 0 {
                if called { 100 } else { 0 }
            } else {
                100 * branch_seen / branch_tot
            }; 
```